assets/css/main.css

```diff
@@ -1,624 +1,693 @@
-  width: 51px;
-  height: 51px;
-  background: transparent;
-  cursor: pointer;
-}
-.menu-toggle .bar {
-  position: absolute;
-  width: 35px;
-  height: 5px;
-  background: #DFDDD7;
-  -webkit-transition: all 300ms ease;
-          transition: all 300ms ease;
-}
-.menu-toggle.after-fold .bar {
-  background: #282928;
-}
-.menu-toggle .bar:nth-child(1) {
-  left: 5px;
-  top: 4px;
-}
-.menu-toggle .bar:nth-child(2) {
-  left: 5px;
-  top: 16px;
-}
-.menu-toggle .bar:nth-child(2):after {
-  content: "";
-  position: absolute;
-  left: 0px;
-  top: 0px;
-  width: 35px;
-  height: 5px;
-  background: #DFDDD7;
-  -webkit-transition: all 300ms ease;
-          transition: all 300ms ease;
-}
-.menu-toggle.after-fold .bar:nth-child(2):after {
-  background: #282928;
-}
-.menu-toggle .bar:nth-child(3) {
-  right: 11px;
-  top: 28px;
-}
-.menu-toggle.open .bar:nth-child(1),
-.menu-toggle.open .bar:nth-child(3) {
-  width: 0;
-}
-.menu-toggle.open .bar:nth-child(2) {
-  -webkit-transform: rotate(-45deg);
-      -ms-transform: rotate(-45deg);
-          transform: rotate(-45deg);
-}
-.menu-toggle.open .bar:nth-child(2):after {
-  -webkit-transform: rotate(-90deg);
-      -ms-transform: rotate(-90deg);
-          transform: rotate(-90deg);
-}
-
-/* END MENU BUTTON */
-
-@media all and (max-width: 770px) {
-  .top-bar h1 {
-    font-size: 1.2rem;
-    margin: 15px 0 15px 20px;
-  }
-  .menu-toggle {
-    display: inline-block;
-  }
-  .header-nav-wrapper {
-    -webkit-flex-basis: 100%;
-        -ms-flex-preferred-size: 100%;
-            flex-basis: 100%;
-    height: auto;
-    display: none;
-  }
-  .header-nav-list {
-    -webkit-flex-basis: 100%;
-        -ms-flex-preferred-size: 100%;
-            flex-basis: 100%;
-    height: auto;
-    padding: 0;
-    -webkit-box-orient: vertical;
-    -webkit-box-direction: normal;
-    -webkit-flex-direction: column;
-        -ms-flex-direction: column;
-            flex-direction: column;
-    margin: 0;
-  }
-  .header-nav-list-item {
-    width: 100%;
-    text-align: center;
-    background: #DFDDD7;
-  }
-  .header-nav-list-item-link {
-    display: inline-block;
-    padding: 15px 0;
-    margin-left: 0;
-    width: 100%;
-    color: #554F4E;
-  }
-  .header-nav-list-item-link:hover {
-    color: #282928;
-  }
-  .header-nav-list-item-link.active {
-    border-bottom: none;
-  }
-  .header-nav-list-item-link.hire-us-link {
-    background-color: #F1853D;
-    color: #fff;
-    padding: 15px 0;
-  }
-}
-
-/* END NAVIGATION */
-
-.splash-image {
-  color: #fff;
-  font-size: 1.35rem;
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-align: center;
-  -webkit-align-items: center;
-      -ms-flex-align: center;
-          align-items: center;
-  min-height: 550px;
-  height: 50vh;
-  background: url("http://res.cloudinary.com/piggybank/image/upload/v1444888085/bridge_hkp6iq.jpg") no-repeat center/cover;
-  background-size: cover;
+.services-blocks{
```

Comment:
here's you're overwriting your divs from inline. This is why I would suggest removing that div base style overwrite up top. Then this would just work and you wouldn't need any of these styles I dont think